src/physical_object.py

```diff
@@ -45,24 +45,80 @@ def get_forces_composition(self):
             force[1] += self.forces[f][1]
         return force
 
+    def reset_forces(self):
+        for f in self.forces:
+            self.forces[f] = [0., 0.]
+
+    def add_force(self, label, force):
+        if label not in self.forces:
```

Comment:
Forces is a dictionary of labels (e.g. 'gravity', 'magnetic', 'friction' etc.). Label is a key in the dictionary.